core/src/main/java/org/commonjava/indy/core/content/PathMaskChecker.java

```diff
@@ -37,26 +37,28 @@ public static boolean checkMask(final ArtifactStore repo, final String path){
             return true;
         }
 
+        String pathForCheck = path.startsWith( "/" ) ? path.substring( 1 ) : path;
+
         for ( String pattern : maskPatterns )
         {
             // adding allPlaintext to the condition to reduce the number of isRegexPattern() calls
             if ( isRegexPattern( pattern ) )
             {
                 final String realRegex = pattern.substring( 2, pattern.length() - 1 );
-                if ( path.matches( realRegex ) )
+                if ( pathForCheck.matches( realRegex ) )
                 {
                     logger.trace( "Checking mask in: {}, pattern with regex: {} - MATCH", repo.getName(), realRegex );
                     return true;
                 }
             }
-            else if ( path.startsWith( pattern ) )
+            else if ( pathForCheck.startsWith( pattern ) )
             {
                 logger.trace( "Checking mask in: {}, pattern: {} - MATCH", repo.getName(), pattern );
                 return true;
             }
         }
 
-        logger.debug( "Path {} not available in path mask {} of repo {}", path, maskPatterns, repo );
+        logger.info( "Path {} not available in path mask {} of repo {}", path, maskPatterns, repo );
```

Comment:
Not sure if it will bring log overflow to set this to info level. Maybe we should still keep it to info? (If we want to print this log, we can use logger api to change the logger level dynamically)

Reply:
Sure, I will revert this back to DEBUG level.